xarray/tests/__init__.py

```diff
@@ -282,7 +296,7 @@ def create_test_data(seed: int | None = None, add_attrs: bool = True) -> Dataset
             obj[v].attrs = {"foo": "variable"}
     obj.coords["numbers"] = (
         "dim3",
-        np.array([0, 1, 2, 0, 0, 1, 1, 2, 2, 3], dtype="int64"),
+        np.random.randint(0, 3, _dims["dim3"], dtype="int64"),
```

Comment:
This change breaks tests in `TestTileIDsFromNestedLists`. Initially I've thought that they are unrelated, but reverting fixes those tests.

Reply:
I reverted it to the previous behavior when default argument is used for dim_sizes. It looks that all tests are passing now : )

Reply:
Sorry, it's me, markelg, I keep getting confused with my two github accounts.